test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -47,9 +47,9 @@ public void AddProductTest(int id, string name, int cost, SupportedLanguages lan
         [SqlInlineData(1, "Test", 5)]
         [SqlInlineData(0, "", 0)]
         [SqlInlineData(-500, "ABCD", 580)]
-        // Currently PowerShell and Java functions return null when the parameter for name is an empty string
-        // Issue link: https://github.com/Azure/azure-functions-sql-extension/issues/443
-        [UnsupportedLanguages(SupportedLanguages.PowerShell, SupportedLanguages.Java)]
+        // Add Support to nullable types in query string to be added in the powershell worker
+        // Currently use a workaround through the TriggerMetadata for PowerShell Azure Functions
+        // Issue link: https://github.com/Azure/azure-functions-powershell-worker/issues/895
```

Comment:
We can just remove this since you already have the comment in the function itself
```suggestion
```